tools/cldr-code/src/main/java/org/unicode/cldr/util/Annotations.java

```diff
@@ -278,11 +319,18 @@ private String getStringValue(String xpath) {
         private String getStringValue(String xpath, CLDRFile cldrFile2, CLDRFile english) {
             String result = cldrFile2.getStringValueWithBailey(xpath);
             if (result == null) {
-                return ENGLISH_MARKER + english.getStringValueWithBailey(xpath);
+                return (ENGLISH_MARKER + english.getStringValueWithBailey(xpath));
             }
             String sourceLocale = cldrFile2.getSourceLocaleID(xpath, null);
-            if (sourceLocale.equals(XMLSource.CODE_FALLBACK_ID) || sourceLocale.equals(XMLSource.ROOT_ID)) {
-                if (!xpath.equals("//ldml/characterLabels/characterLabelPattern[@type=\"category-list\"]")) {
+            if (
+                sourceLocale.equals(XMLSource.CODE_FALLBACK_ID) ||
+                sourceLocale.equals(XMLSource.ROOT_ID)
+            ) {
+                if (
+                    !xpath.equals(
```

Comment:
In general, I find it harder to read, because it expands to too many lines.

Reply:
one way to address that, in this case, might be to assign the string literal "//ldml/characterLabels/characterLabelPattern[@type=\"category-list\"]" to a constant first

Reply:
again, see https://github.com/unicode-org/cldr/pull/1919 which leaves this line unchanged